notebooks/pudl-catalog.ipynb

```diff
@@ -6,7 +6,27 @@
     "tags": []
```

Comment:
<p>Maybe just to be totally clear about the environment variables I think the bullet points could be rearranged to read something like this:</p><p>- You need to set the <code>PUDL_INTAKE_CACHE</code> and <code>PUDL_INTAKE_PATH</code> environment variables</p><p>- That can be done either within the environment that you're running Jupyter in, or in the notebook itself in the cell below, prior to the <code>import intake</code> line.&nbsp;(also there's a typo in this bullet point above, "runing" --&gt; "running")</p><p>- If you do need custom environment variable values, they need to be set before the <code>intake</code> package is imported by Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl-catalog/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Okay, I've clarified the environment variable stuff.</p>